docs/end-user/filtering-call-trees.md

```diff
@@ -0,0 +1,81 @@
+
+The implementation filter will modify the shape of stacks, and generate a completely new call tree. If any of the filtered stacks are empty, then those samples will be dropped from the analysis.
+
+## Invert call stack
+
+![A screenshot of the "Invert call stacks" checkbox.](../images/invert-2018-05.jpg)
+
+Inverting the call stacks of samples will produce an entirely new call tree. All of the self time will be at the roots of the call tree. The implications to the shape of the call tree may be surprising, but consider the following graphics.
+
+![An image showing samples on the left, and the call tree on the right. This is before the invert operation.](../images/invert-before.svg)
+
+In this profile, for the sake of brevity, there are only 3 samples that were collected 1ms apart. In most uninverted call trees, there is only one root node. In this case `A` calls `B`, and it's not until the third stack frames where the calls diverge to function `X` and function `C`. The call tree on the right is what is produced from these samples. Now consider inverting all of the stacks in the samples.
+
+![An image showing samples on the left, and the call tree on the right. This is after the invert operation.](../images/invert-after.svg)
+
+Previously, the stacks `X -> Y -> Z` were each at different levels. However, in the inverted view, the stacks are now at the same level. When the call tree is created from these samples, there are now two different roots, starting from the function `Z` and the function `E`. The inversion made the `Z -> Y -> X` portions of the stacks fall into the same level.
+
+Inversion is most useful in surfacing the functions where time is actually being spent. The trees are typically much larger and noisier than uninverted call trees.
+
+## Transforms
+
+Call tree transforms provide a finer grain control over how to modify the call tree. They modify the stack in different operations. They can operate at individual call nodes on the tree, or across the entire tree for a given function. Different programs use different terms to describe these operations, but perf.html has defined certain operations.
+
+### Merge
+
+Merging takes a call node, and removes it from the call tree. Any self time for that node is then charged to its parent node. This can be done for a single call node, or for all functions in the tree. This type of operation can be useful for modifying the shape of the tree, and removing unhelpful functions from the call tree.
+
+### Focus
+
+Focusing on a function or call node removes all of the ancestor call nodes—the children call nodes remain. If a stack does not contain that function or node, then it is removed. This effectively focuses on a subtree or a set of subtrees on the call tree.
```

Comment:
Optional nit: I'd use some space around the emdash (`call nodes - the children`) as the font github uses crushes it too much against the text.  (same in other places where you use an emdash)